docs/plugins/markdown/prismjs.md

```diff
@@ -194,6 +194,207 @@ export default defineUserConfig({
 })
 ```
 
+### collapsedLines
+
+- Type: `boolean | number | 'disabled'`
+
+- Default: `'disabled'`
+
+- Details: Whether to enable code block collapsing.
+
+  - `number`: collapse the code block starting from line `number`, for example, `12` means collapsing the code block starting from line 12.
+  - `true`: Equivalent to `15`, collapsing the code block starting from line 15.
+  - `false`: Disable code block collapsed, but `:collapsed-lines` can still be added to collapse individual code blocks.
```

Comment:
I think the description can be changed to: 

Add support for code block collapsing,  but disable it globally

Also I would prefer the word `code block collapsing` instead of code block collapsed.